### DISCUSSION TITLE
Need help with Satin stitches.

### DISCUSSION THREAD

**Discussion by KWhitt92** [inkstitch/inkstitch#2393] [Q&A]
I am working on a satin stitch design. Everything in Ink/Stitch looks beautiful! But when stitching, or viewing in a different embroidery program something goes terribly wrong. Most of the satin stitches disappear and are replaced with running stitches! This is the second time I have attempted this file. And the first attempt to stitch had the opposite happen. Main part stayed satin and the tops and bottoms switched to running stitches. 
![Untitled](https://github.com/inkstitch/inkstitch/assets/137945011/aca2da1a-d686-4532-8b56-4cf3ae2a6db6)
![Untitled2](https://github.com/inkstitch/inkstitch/assets/137945011/7155ff89-8e65-412a-8f65-65c77a05b72d)
![Untitled3](https://github.com/inkstitch/inkstitch/assets/137945011/8cadf6f2-b1c9-4055-b7bb-eb50b4c3d2b5)
![354880276_253640827291326_1329085784394118331_n](https://github.com/inkstitch/inkstitch/assets/137945011/86a99f78-e290-4ac6-b0e5-39c61c93ee2b)

**Comment by kaalleen** [comment]
Can you attach the svg file? Even if you fixed the issue for yourself already. The svg file can still be helpful to fix Ink/Stitch and prevent that others run into the same issue at a later time. 

Other than that, have a look at various options to design sharp edges in embroidery files: https://inkstitch.org/tutorials/satin-edges/
